README.md

```diff
@@ -202,6 +202,16 @@ Documentation for using each command is available by typing:
 > python manage.py help <command>
 ```
 
+### Pickle Tribe gene sets
+
+This step downloads and pickles all the public Tribe gene sets for every
+organism that has been loaded into the database. These pickled files are
+necessary for doing gene set enrichment analyses.
```

Comment:
Oh, gotcha. I forgot that other parts of this page have instructions on installing `tribe_client` already. Hmm... a short link wouldn't hurt though. Sometimes I search documentation and jump into the middle somewhere. In a case like that it's handy to have a cross reference right where I need it. Something short like "for more information, see the tribe_client documentation," with an embedded link, maybe? Up to you.

Bumping requirements.txt would be nice. Good idea.